projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -619,10 +619,25 @@ public function allow_feedback_rest_api_type( $post_types ) {
 	 * @param object $screen Information about the current screen.
 	 */
 	public function unread_count( $screen ) {
-		if ( isset( $screen->post_type ) && 'feedback' === $screen->post_type ) {
+		if ( isset( $screen->post_type ) && 'feedback' === $screen->post_type || $screen->id === 'jetpack_page_jetpack-forms-admin' ) {
 			update_option( 'feedback_unread_count', 0 );
 		} else {
 			global $submenu;
+			if ( apply_filters( 'jetpack_forms_use_new_menu_parent', true ) ) {
+				// show the count on Jetpack → Forms
+				$unread = get_option( 'feedback_unread_count', 0 );
+
+				if ( $unread > 0 && isset( $submenu['jetpack'] ) && is_array( $submenu['jetpack'] ) && ! empty( $submenu['jetpack'] ) ) {
+					foreach ( $submenu['jetpack'] as $index => $menu_item ) {
+						if ( 'jetpack-forms-admin' === $menu_item[2] ) {
+							$unread_count = current_user_can( 'publish_pages' ) ? " <span class='feedback-unread count-{$unread} awaiting-mod'><span class='feedback-unread-count'>" . number_format_i18n( $unread ) . '</span></span>' : '';
```

Comment:
Why is the unread_count dependent on `current_user_can( 'publish_pages' )` permission? 
Do we only want to show it to some users?

Reply:
I'd think since we're adding this new code for our new page, we can probably remove the current_user_can( 'publish_pages' ) if we don't see a good reason for it to be there?

Reply:
I'm seeing similar restrictions on other places: Admin bar, feedback custom post type definition. I'd say let's maybe leave it there, but create a follow up task to review all our capabilities checks. It wouldn't make sense to show the badge if the user can't access the dashboard.
As far as I saw, we check for:
- `current_user_can( 'edit_pages' )`
- `current_user_can( 'publish_pages' )` (current chunk)
- `current_user_can( 'edit_pages' )` (form creation link)
- `current_user_can( 'edit_pages' )` (API endpoint)

So, I'd say let's change this one to `edit_pages` as well so at least we have consistency? But also, perform the check even before doing anything there.